src/get-started/install.md

```diff
@@ -66,6 +66,24 @@ with its compilation in certain configurations. If you have encountered a proble
 with installing Exonum on Windows, you may ask around on [Gitter](https://gitter.im/exonum/exonum)
 or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
+## Adding environment variables
+
+This variables needed for linking with the `rocksdb` and `snappy` libraries.
```

Comment:
Language: "variables *are* needed"? Plus, they are not really required, at least from my experience; I didn't set these variables (more than that, one of my envs in Windows 8.1), but Exonum works just fine :thinking:

Reply:
Setting these vars improves compile times, afaik. Btw, @SerhiiOryshych, `these`, not `this`